unabridged-works-volume-1/original/03-way-of-life.adoc

```diff
@@ -630,13 +641,15 @@ And this is a grievous apostasy, and the root, spring, and foundation of all the
 for he that misseth in his beginning, he that begins his religion without the true rule,
 how can he proceed aright in any thing afterwards?
 
-Objection.
+[.discourse-part]
+_Objection._
```

Comment:
Hmmm... so i looked into it a bit, and my "auto-italicization" feature for `.discourse-part`s was not working on these, because I think it was designed to only recognize when it has a colon instead of a period.  So, no need to change these back, but going forward, if the DP starts with Question, Answer, Objection, or Inquiry -- leave it _not_ italicized.  I'm going to fix it on my end so these end up properly italicized in the output, without having to make it that way in the asciidoc.